### DISCUSSION TITLE
404 page when i18n is enabled

### DISCUSSION THREAD

**Discussion by heesungjang** [shuding/nextra#779] [Q&A]
I tried to follow the steps to enable i18n on nextra doc,

however,

locale prefix only works for meta.json file (e.g. meta.ko.json works), but when I add a prefix to index.mdx files like index.ko.mdx and index.en.mdx, it shows 404 page.

Are there any other configs that I have to do besides the steps explained on the docs page?

thanks!

**Comment by dimaMachina** [comment]
Do you followed swr example? You must configure middleware as well

**Reply by heesungjang** [reply]
@B2o5T cool! thks for the reply!! will go and try 👍

**Reply by heesungjang** [reply]
@B2o5T it does work with nextra1 but not with nextra 2 beta. Is it not supported yet?

**Reply by dimaMachina** [reply]
Follow this example https://github.com/shuding/nextra/tree/core/examples/swr-site

**Reply by heesungjang** [reply]
legend! thank you so much for your help @B2o5T.

**Comment by dimaMachina** [comment]
https://nextra.site/docs/guide/i18n#custom-404-page-optional
